### PR TITLE
alerts and error styles

### LINKED ISSUE #229: Add form validation and error styles
v1 - on form submission. Borrowing from [bootstrap](http://getbootstrap.com/css/#forms-control-validation) 
Mention and show important ones.

### PR DESCRIPTION
**Pull request by binx**
closes #229

### CODE REVIEW

File: src/stylesheets/common/_alerts.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+.alert-info {
+  background-color: rgba(56,93,92,.1);
+  border-color: rgba(56,93,92,.3);
+  color: rgba(56,93,92,.8);
+}
+.alert-warning {
+  background-color: rgba(211,199,86,.2);
+  border-color: rgba(211,199,86,.6);
+  color: #91893B;
```
Comment (hanbyul-here) [inline src/stylesheets/common/_alerts.scss]: do you have reason for mixing hex value and rgb value?

**Comment by binx** [comment]
@hanbyul-here the rgb values were taken from the styleguide and the hex values were new colors. I can convert them all to rgba if you'd like.

Although, that would still be mixing rgb and rgba values, unless I figure out an rgb value that looks like #91893B at 0.8 opacity

**Comment by hanbyul-here** [comment]
No I didn't mean that. I am not worried that much about mixing rgb and hex value in whole styleguide, but mixing hex and rgba inside of one element ( `alert-warning` , the line that I commented on) can be little confusing later? maybe I am overthinking.

**Comment by binx** [comment]
I was talking about the same thing :)
I'll switch it out.

**Pull request merged**